CHANGELOG.md

```diff
@@ -8,6 +8,25 @@ Remember to align the itemized text with the first line of an item within a list
 
 ## jax 0.4.8
 
+* Breaking changes
+  * A major component of the Cloud TPU runtime has been upgraded. This enables
+    the following new features on Cloud TPU:
+    * `jax.debug.print`, `jax.debug.callback`, and `jax.debug.breakpoint()` now
```

Comment:
Optional: use e.g. ``` {func}`jax.debug.print` ``` so the generated HTML will include links to the function documentation

Reply:
Good idea, done!